registry/quilt_server/const.py

```diff
@@ -20,3 +20,11 @@ class PaymentPlan(Enum):
     TEAM_UNPAID = 'team_unpaid'
 
 FTS_LANGUAGE = 'english'
+
+BAD_NAMES = set([
```

Comment:
Why are these upper case? We should probably add to this list. In Django we also had: quilt and anonymous.